litho-it/src/test/java/com/facebook/litho/specmodels/generator/BuilderGeneratorTest.java

```diff
@@ -63,13 +63,23 @@ public void testEventMethod(@Prop boolean arg0) {}
     public void testUpdateStateMethod() {}
   }
 
+  @LayoutSpec
+  static class TestResTypeWithVarArgsSpec {
+    @OnCreateLayout
+    public void test(@Prop(varArg = "test", resType = ResType.DIMEN_TEXT) List<Float> arg0) {}
```

Comment:
Could you use some more descriptive names here? `test` twice makes this a bit difficult to follow.

Reply:
Now the `varArg` names `size` and the argument names `sizes` maybe it's easier to follow